### PR TITLE
azure ad configuration

### PR DESCRIPTION
**Pull request by SANTOSH-MWP**
[https://mavenwave.atlassian.net/browse/MC-186](url)

### CODE REVIEW

File: app-config.yaml [hunk 1]
```diff
@@ -43,19 +39,7 @@ backend:
...
-    #   user: postgres # ${POSTGRES_USER}
-    #   password: secret # ${POSTGRES_PASSWORD}
-    # plugin:
-    #   catalog:
-    #     connection:
-    #       database: postgres
-    # app:
-    #   connection:
-    #     database: postgres
+
```
Comment (fosterm-mw) [inline app-config.yaml]: I am going to leave these uncommented out, other than that looks good.

**Review by fosterm-mw — COMMENTED** [review]
I ran locally and did not have the groups or users loaded into backstage. Maybe I'm doing something wrong, we can discuss over chat.

**Comment by SANTOSH-MWP** [comment]
> 

pls update the app API permission in AD , whichever is in use. 
e.g
GroupMember read
User read

**Review by fosterm-mw — APPROVED** [review]
good now

**Pull request merged**